plugin-manager/src/main/kotlin/com/itsaky/androidide/plugins/manager/loaders/PluginLoader.kt

```
@@ -249,6 +248,24 @@ class PluginLoader(
+            val pm = context.packageManager
+            @Suppress("DEPRECATION")
+            val info = pm.getPackageArchiveInfo(
+                pluginApk.absolutePath,
+                PackageManager.GET_SIGNING_CERTIFICATES or PackageManager.GET_SIGNATURES
+            )
+            @Suppress("DEPRECATION")
+            val legacySignatures = info?.signatures
+            val signatures = info?.signingInfo?.apkContentsSigners?.takeIf { it.isNotEmpty() }
+                ?: legacySignatures
+            signatures?.firstOrNull()?.toByteArray()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)